amo2kinto/synchronize.py

```diff
@@ -14,13 +28,23 @@ def get_diff(source, dest):
     dest_keys = set(dest_dict.keys())
     to_create = source_keys - dest_keys
     to_delete = dest_keys - source_keys
+    to_update = set()
+
+    to_check = source_keys - to_create - to_delete
+
+    for record_id in to_check:
+        new = canonical_json(source_dict[record_id])
+        old = canonical_json(dest_dict[record_id])
+        if new != old:
```

Comment:
There are no such thing as enabled in the source record, that's why we strip it because we want to compare what is in both side only.